src/shared/proto/geometry.proto

```diff
@@ -3,23 +3,29 @@ syntax = "proto3";
 message PointMsg
 {
     // x coordinate
-    double x = 1;
+    float x_meters = 1;
```

Comment:
Not sure if it makes sense for a point to have a unit? The unit is interpreted by the thing using Point. 

But don't feel strongly about this, because we generally only use Points for things on the field which is in meters.

Reply:
hm how can we make this consistent with Angle, which is in radians?

Reply:
yeah thats a fair point, don't want to block this PR on this, we can revisit it if it becomes a problem later